src/components/Footer/Footer.module.css

```diff
@@ -2,16 +2,17 @@
 .footerWrap {
     width: 90%;
     min-height: 30px;
+    margin-bottom: 10px;
     padding: 5px 10px;
     display: flex;
+    justify-content: space-around;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/alde_react/pullRequest?prid=5576104)